windows/sound.cpp

```diff
@@ -143,9 +144,9 @@ QString CSound::CheckDeviceCapabilities()
     if ( ( CanSaRateReturn == ASE_NoClock ) || ( CanSaRateReturn == ASE_NotPresent ) )
     {
         // return error string
-        return tr ( "The audio device does not support the "
-                    "required sample rate. The required sample rate is: " ) +
-               QString().setNum ( SYSTEM_SAMPLE_RATE_HZ ) + " Hz";
+        return QString ( tr ( "The selected audio device is incompatible "

```

Comment:
```suggestion
        return QString ( tr ( "The selected sound hardware is incompatible "

```

Reply:
So here, would we consider using device after all? As per https://github.com/jamulussoftware/jamulus/pull/1732#discussion_r637388198